test/unit/dashboard/launch.test.js

```diff
@@ -0,0 +1,69 @@
+const expect = require('chai').expect;
+const io = require('socket.io-client');
+const { nanoid } = require('nanoid');
+
+const id = nanoid(16);
+
+// unit under test
+const Server = require('../../../dashboard/server');
```

Comment:
If you're testing `dashboard/server`, this file should be `test/unit/dashboard/server.test.js`